rotkehlchen/tests/unit/accounting/evm/test_balancer.py

```diff
@@ -217,3 +350,303 @@ def test_balancer_multiple_deposit(accountant: 'Accountant', ethereum_inquirer:
+                type=AccountingEventType.TRANSACTION_EVENT,
+                notes='Deposit 131421.703252 USDC to a Balancer v1 pool',
+                location=Location.ETHEREUM,
+                timestamp=TIMESTAMP_1_SECS,
+                asset=A_USDC,
+                free_amount=ZERO,
+                taxable_amount=FVal('131421.703252'),
+                price=Price(ONE),
+                pnl=PNL(taxable=ZERO, free=ZERO),  # Deposits are not taxable
+                cost_basis=None,
+                index=1,
+                extra_data={'tx_hash': EVM_HASH.hex()},  # pylint: disable=no-member
+            ),
+            ProcessedAccountingEvent(
+                type=AccountingEventType.TRANSACTION_EVENT,
+                notes='Refunded 6578.947368421052624578 DAI after depositing in Balancer V1 pool',
+                location=Location.ETHEREUM,
+                timestamp=TIMESTAMP_1_SECS,
+                asset=A_DAI,
+                free_amount=ZERO,
+                taxable_amount=FVal('6578.947368421052624578'),
+                price=Price(ONE),
+                pnl=PNL(taxable=ZERO, free=ZERO),  # Withdrawals/refunds are not taxable
+                cost_basis=None,
+                index=2,
+                extra_data={'tx_hash': EVM_HASH.hex()},  # pylint: disable=no-member
+            ),
+            ProcessedAccountingEvent(
+                type=AccountingEventType.TRANSACTION_EVENT,
+                notes='Refunded 6571.085163 USDC after depositing in Balancer V1 pool',
+                location=Location.ETHEREUM,
+                timestamp=TIMESTAMP_1_SECS,
+                asset=A_USDC,
+                free_amount=ZERO,
+                taxable_amount=FVal('6571.085163'),
+                price=Price(ONE),
+                pnl=PNL(taxable=ZERO, free=ZERO),  # Withdrawals/refunds are not taxable
+                cost_basis=None,
+                index=3,
+                extra_data={'tx_hash': EVM_HASH.hex()},  # pylint: disable=no-member
+            ),
+            ProcessedAccountingEvent(
+                type=AccountingEventType.TRANSACTION_EVENT,
+                notes='Receive 1675.495956074927519908 BPT from a Balancer v1 pool',
+                location=Location.ETHEREUM,
+                timestamp=TIMESTAMP_1_SECS,
+                asset=Asset('eip155:1/erc20:0x72Cd8f4504941Bf8c5a21d1Fd83A96499FD71d2C'),
+                free_amount=FVal('1675.495956074927519908'),
+                taxable_amount=ZERO,
+                price=Price(ONE),
+                pnl=PNL(taxable=ZERO, free=ZERO),  # Deposits are not taxable
+                cost_basis=None,
+                index=4,
+                extra_data={'tx_hash': EVM_HASH.hex()},  # pylint: disable=no-member
+            ),
+            ProcessedAccountingEvent(
+                type=AccountingEventType.TRANSACTION_EVENT,
+                notes='Lost 0.000000000000000015 from withdrawing 124999.999999999999867 DAI from balancer-v1 pool',  # noqa: E501
```

Comment:
AS we discussed in discord I believe we should account for the total USD value deposited (the sum of the assets deposited) and then the USD value of the withdrawn assets. This way if you deposit a lot of DAI and withdraw USDC you can evaluate the PnL